packages/composites/savax-price/README.md

```diff
@@ -0,0 +1,45 @@
+# Chainlink Savax-price Composite Adapter
+
+This External Adapter pulls the price of `USD/sAVAX`. It exists as a composite EA as
```

Comment:
Incomplete sentence, what was this meant to say?

Reply:
fixed this